spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -428,33 +428,33 @@ def assert_specific_vm
     expect(nic.network).to eq(network)
 
     expect(v.parent_datacenter).to have_attributes(
-      :ems_ref       => "datacenter-2",
-      :ems_ref_obj   => VimString.new("datacenter-2", :Datacenter, :ManagedObjectReference),
-      :uid_ems       => "datacenter-2",
-      :name          => "Prod",
-      :is_datacenter => true,
+      :ems_ref     => "datacenter-2",
+      :ems_ref_obj => VimString.new("datacenter-2", :Datacenter, :ManagedObjectReference),
+      :uid_ems     => "datacenter-2",
+      :name        => "Prod",
+      :type        => "Datacenter",
 
-      :folder_path   => "Datacenters/Prod"
+      :folder_path => "Datacenters/Prod"
     )
 
     expect(v.parent_folder).to have_attributes(
-      :ems_ref       => "group-d1",
-      :ems_ref_obj   => VimString.new("group-d1", :Folder, :ManagedObjectReference),
-      :uid_ems       => "group-d1",
-      :name          => "Datacenters",
-      :is_datacenter => false,
+      :ems_ref     => "group-d1",
+      :ems_ref_obj => VimString.new("group-d1", :Folder, :ManagedObjectReference),
+      :uid_ems     => "group-d1",
+      :name        => "Datacenters",
+      :type        => nil,
```

Comment:
@Fryguy and I looked at that earlier, it seems that `:type => nil` co-implies that it is the base class but he was surprised too.  Trying this on a few other STI models (like VmOrTemplate) showed the same thing.